src/tir/usmp/analysis/extract_buffer_info.cc

```diff
@@ -491,6 +559,55 @@ BufferInfoAnalysis BufferInfoExtractor::operator()(const PrimFunc& main_func) {
       open_set.erase(le_event.buffer_info);
     }
   }
+
+  // All items in RO pool should have conflicts with each other in this RO pool
+  // as they will be placed in RO segment and pre-initialized
+  // Array<BufferInfo> buffer_info_arr =
+  //    ConvertToArrayOfBufferInfo(this->buffer_info_map_);
+
+  // split buffers to vars (RW) and constants (RO)
+  Array<BufferInfo> buffer_info_vars;
+  Array<BufferInfo> buffer_info_cons;
```

Comment:
nit : may be using buffer_info_constants is a better name ?

Reply:
renamed